webapp/src/pages/Settings.tsx

```diff
@@ -548,8 +569,38 @@ const Settings: React.FC = () => {
     </FormGroup>
   );
 
+  const displayAnalysesCustomizationGeneralSection = () => (
+    <Box display="flex" justifyContent="flex-start" gap={5} alignItems="center">
+      <FormControl variant="standard" sx={{ m: 1, width: 120 }}>
```

Comment:
A couple of things here:
1. The `justifyContent="flex-start"` doesn't seem to have any effect, can we remove it?
1. Like the other sections, this one should be wrapped inside a `<FormGroup>` to give it the standard margin. You can then remove the `m: 1` on `FormControl`.
1. Let's give the input the same width as the number fields, for consistency. This is done via a class name, currently called `number`, but that we could rename to something like `fixedWidthInput`. You can then remove the `width: 120` on `FormControl`.

```diff
       variant="standard"
       size="small"
       type="number"
-      className="number"
+      className="fixedWidthInput"
```
```diff
-          [`& .number .${inputClasses.root}`]: { width: "12ch" },
+          [`& .fixedWidthInput .${inputClasses.root}`]: { width: "12ch" },
```

```suggestion
    <FormGroup>
      <Box display="flex" gap={5} alignItems="center">
        <FormControl variant="standard" className="fixedWidthInput">
```